grr/server/grr_response_server/gui/api_plugins/metadata.py

```diff
@@ -662,12 +831,27 @@ def _GetPathArgsFromPath(path: str) -> List[str]:
 
 
 def _GetTypeName(
-    cls: Union[Descriptor, FieldDescriptor, EnumDescriptor, Type, int, str],
+    cls: Optional[
+      Union[Descriptor, FieldDescriptor, EnumDescriptor, Type, int, str]
+    ],
 ) -> str:
   """Extract type name from protobuf `Descriptor`/`type`/`int`/`str`."""
   if isinstance(cls, FieldDescriptor):
+    if _IsMapField(cls):
+      map_type_name = _GetTypeName(cls.message_type)
+      if map_type_name.endswith("Entry"):
+        map_type_name = map_type_name[:-5]
+
+      key_type_d, value_type_d = _GetMapFieldKeyValueTypes(cls)
+
+      return (
+        f"{map_type_name}Map_"
+        f"{_GetTypeName(key_type_d)}:{_GetTypeName(value_type_d)}"
+      )
```

Comment:
I would put results of `_GetTypeName` and make the string literal fit one line, right now it is hard to parse.

Reply:
I agree, it's not very readable. I made the change. Thank you!